spec/support/page_objects/publish/courses/edit_engineers_teach_physics.rb

```diff
@@ -8,7 +8,7 @@ class EditEngineersTeachPhysics < PageObjects::Base
 
         section :campaign_fields, '[data-qa="course__engineers_teach_physics_radio_group"]' do
           element :engineers_teach_physics, '[data-qa="course__campaign_name_engineers_teach_physics_radio"]'
-          element :no_campaign, '[data-qa="course__campaign_name_no_campaign_radio"]'
+          element :the_no_campaign, '[data-qa="course__campaign_name_no_campaign_radio"]'
```

Comment:
Can't think of a better name for this, but it might not be obvious why there is a `the_` there. The enum name is `campaign_name` so maybe `name_no_campaign`. But then there isn't a pattern followed by the other elements.